src/components/CertificateVerifyBlock/DetailedCertificateVerifyBlock/DetaledCertificateVerifyBlock.test.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { act } from "react-dom/test-utils";
+import DetailedCertificateVerifyBlock from ".";
+import ROPSTEN from "../../HomePageContent/Ropsten-Demo.json";
+import TokenRegistry from "../../../test/fixture/tokenRegistry.json";
```

Comment:
```suggestion
import sampleToken from "../../../test/fixture/sample-token.json";
```

Have created an issue to rename the file, we can do it here or in another PR

Reply:
Renamed the file and respective imports.